.github/workflows/build.yml

```diff
@@ -29,6 +29,8 @@ jobs:
         run: cs install --channel channel bleep --verbose
 
       - name: Run tests
+        env:
+          CI: true
```

Comment:
Isn't this redundant? :thinking: 
https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables=

Reply:
Yeah, that's probably why I deleted the declarations in the first place, heh :)

I was trying to get the first branch taken [here](https://github.com/oyvindberg/bleep/blob/master/bleep-cli-test/src/scala/bleep/IntegrationSnapshotTests.scala#L24-L26), but it somehow still didn't work. I just merged this branch without digging too much, as it'll need to be revisited very soon anyway.

The integration snapshot tests imports a bunch of real, complex sbt builds into bleep, and makes a bunch of property tests on the result. Currently all these other builds are git submodules which need to be checked out for every build, and coursier needs to resolve all the dependencies for all the builds. as a result the github action cache is absolutely enormous. My plan is to replace submodules with code which checks out the projects when needed, and try to wrestle coursier into resolving dependencies without actually downloading them.